_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -26,7 +26,8 @@ This rule applies to any HTML `input`, `select` and `textarea` element with an `
 - **hidden**: the element is not [visible][], and not [included in the accessibility tree][]; or
 - **disabled**: the element is a [disabled element]; or
 - **fixed value**: the element is an `input` element with a `type` [attribute value][] of either `hidden`, `button`, `submit` or `reset`; or
-- **static**: the element is not part of [sequential focus navigation][] and has a [semantic role][] that is not a [widget role][].
+- **static**: the element is not part of [sequential focus navigation][] and has a [semantic role][] that is not a [widget role][]
+- the `autocomplete` attribute wears the [autofill expectation mantle](https://html.spec.whatwg.org/#autofill-expectation-mantle).
```

Comment:
Also, this is a list of exception. Technically, this is currently saying that the rule does **not** apply when the autofill expectation mantle is on and thus **only** apply when the autofill anchor mantle is on. Combined with the **fixed value** condition that excludes the only case where the autofill anchor mantle is on, this means nothing is applicable (a negation is probably needed somewhere…)

Reply:
Ohh, my 🤦 
The idea was that this rule should not apply to `autocomplete` attribute that wears the autofill expectation mantle that is a single token of either `off` or `on` and not an autofill detail token. Will get that sorted now. Thanks!